sdk/containerservice/Azure.ResourceManager.ContainerService/api/Azure.ResourceManager.ContainerService.netstandard2.0.cs

```diff
@@ -462,6 +469,14 @@ public partial class AgentPoolUpgradeProfilePropertiesUpgradesItem
         public static bool operator !=(Azure.ResourceManager.ContainerService.Models.AutoScaleExpander left, Azure.ResourceManager.ContainerService.Models.AutoScaleExpander right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class AzureKeyVaultKms
```

Comment:
Add a prefix?

Reply:
Should avoid using `Azure` as prefix

Reply:
Renamed to `ManagedClusterSecurityProfileKeyVaultKms`